packages/salesforcedx-utils-vscode/test/jest/context/workspaceContextUtil.test.ts

```diff
@@ -52,16 +56,18 @@ describe('WorkspaceContextUtil', () => {
       'clearInstance'
     );
 
-    mockFileSystemWatcher = (vscode.workspace
-      .createFileSystemWatcher as any).mockReturnValue(mockWatcher);
+    mockFileSystemWatcher = mockedVSCode.workspace
```

Comment:
if we use the `jest.mocked` function we can add return/resolve calls to known mocked modules.